src/components/protection/AddProtectionSingle.vue

```diff
@@ -416,39 +415,27 @@ export default class AddProtectionSingle extends BaseComponent {
     });
   }
 
-  async loadAvailableSpace() {
-    this.loadingAvailableSpace = true;
-    this.amountToMakeSpace = await vxm.ethBancor.getAvailableSpace({
-      poolId: this.pool.id
-    });
-    this.loadingAvailableSpace = false;
-  }
-
-  async loadMaxStakes() {
-    if (this.loadingMaxStakes) return;
-    this.loadingMaxStakes = true;
-    try {
-      const result = await vxm.ethBancor.getMaxStakesView({
-        poolId: this.pool.id
-      });
-      result.forEach(async x => {
-        if (x.token === this.token.symbol) {
-          this.maxStakeAmount = x.amount;
-          this.maxStakeSymbol = x.token;
-        }
-      });
-    } catch (e) {
-      console.log(e);
-    } finally {
-      this.loadingMaxStakes = false;
-    }
-  }
+  async load() {
+    if (this.loading) return;
+    this.loading = true;
 
-  async fetchTknSpaceAvailable() {
-    const result = await vxm.ethBancor.getMaxStakesView({
+    const res = await vxm.ethBancor.getAvailableAndAmountToGetSpace({
```

Comment:
Wrap this call in a try catch block, should this throw for whatever reason it's not going to reach things like ```this.loading = false```